src/main/kotlin/build/buf/gradle/OutputSupport.kt

```diff
@@ -20,6 +20,7 @@ import java.io.File
 
 const val BUF_BUILD_DIR = "bufbuild"
 
+@Suppress("DEPRECATION") // https://docs.gradle.org/current/userguide/upgrading_version_8.html#deprecations_3
```

Comment:
I think migrating off `buildDir` to the newer APIs would be good but that might break users of the plugin on older Gradle versions. I think we can postpone this until Gradle 9.x or later?